contracts/protocore/SelfProtocore.sol

```diff
@@ -19,12 +19,17 @@ import "../protocore/GenesisSelfProtocore.sol";
 import "../proxies/MasterCopyNonUpgradable.sol";
 import "../validator/ValidatorSet.sol";
 
-
 /**
  * @title Self protocore - This contract finalizes the proposed blocks of auxiliary chain.
  */
 contract SelfProtocore is MasterCopyNonUpgradable, GenesisSelfProtocore, Protocore, ValidatorSet {
 
+    /* Storage */
+
+    /** Current dynasty */
+    uint256 public dynasty;
```

Comment:
We do not need to track `dynasty` of a SelfProtocore.

Reply:
Yesterday we discussed that it will be needed for the self protocol.
I can remove this.

Reply:
We can keep genesisDynasty in GenesisSelfProtocore as a record of the previous state, however, in this stage we do not need to have it (dynasty) in SelfProtocore. It's not going to be used in any way which is strange.

Reply:
Removed.